### DISCUSSION TITLE
Hydra process monitoring via Dynatrace

### DISCUSSION THREAD

**Discussion by eeraser710** [ory/hydra#2698] [Q&A] [answered]
Dynatrace show below message after enabling hydra process to be monitored via dynatrace . 
 how below can be addressed and resolved ? 

Message:-
![image](https://user-images.githubusercontent.com/18215245/130962977-c6b0d8ac-7ac6-478d-8bb8-7f1973db0336.png)

Hydra version used :- hydra-1.1.0 & hydra-1.10.2
Platform :- Centos 7 ( 64-bit)

As per documentation of dynatrace :- 
![image](https://user-images.githubusercontent.com/18215245/130963115-a48ac236-b809-4e4f-ae0c-d51094b28ff8.png)

related links for reference :

    https://www.dynatrace.com/support/help/technology-support/application-software/go/#application-binary-must-be-dynamically-linked

    https://www.dynatrace.com/support/help/technology-support/application-software/go/support/go-known-limitations/#expand-example-347

**Comment by vinckr** [comment, marked as answer]
Sorry for the late reply @eeraser710! 

How are you running Hydra? Docker, macOs, built from source, etc...?

Here is a good overview of the supported tracing for Ory Hydra:
https://www.ory.sh/hydra/docs/guides/tracing/
We are looking to support all opentracing compliant backends: https://opentracing.io/docs/supported-tracers/, but Dynatrace is not on the radar at the moment (as far as I know). 

Apologies not overly familiar with the issue&dynatrace, but I will see if I can find out some more.

**Reply by vinckr** [reply]
Hey @eeraser710 , just checking in if your problem still persists. 
Sorry I have not been able to find out more regarding Dynatrace.
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
